tests/prepare.ts

```diff
@@ -110,9 +110,6 @@ const log = async <T>(label: string, fn: () => Promise<T>) => {
           middleware: 'koa',
           routesDir: './fixtures/koa',
         },
-        undefined,
-        undefined,
-        metadata,
```

Comment:
What makes this change necessary?

Reply:
For some reason, now that I used a different controller for each framework, the `koa` generated `routes.ts` file kept having the `MiddlewareExpressController` imported instead of `MiddlewareKoaController`.
I figured it must be something about those `metadata` (the `tsoa.json` file used to generate those is configured for the `express` server), so I tried removing those, and the tests suddenly passed..